docs/partial-inputs.ipynb

```diff
@@ -0,0 +1,446 @@
+   "source": [
+    "outdir = Path(\"data/minicubes\")\n",
+    "assert outdir.exists()\n",
+    "\n",
+    "# Write tile to output dir\n",
+    "for tile in stack:\n",
+    "    # Grid code like MGRS-29SNB\n",
+    "    mgrs = str(tile.coords[\"grid:code\"].values).split(\"-\")[1]\n",
+    "    date = str(tile.time.values)[:10]\n",
+    "\n",
+    "    name = \"{dir}/claytile_{mgrs}_{date}.tif\".format(\n",
+    "        dir=outdir,\n",
+    "        mgrs=mgrs,\n",
+    "        date=date.replace(\"-\", \"\"),\n",
+    "    )\n",
+    "    tile.rio.to_raster(name, compress=\"deflate\")\n",
+    "\n",
+    "    with rasterio.open(name, \"r+\") as rst:\n",
+    "        rst.update_tags(date=date)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebc4b6ee-db58-4005-9689-a7d0acdc6a79",
+   "metadata": {
+    "scrolled": true
+   },
+   "source": [
+    "## Create embeddings\n",
+    "Now switch gears and load the tiles to create embeddings and analyze them. \n",
+    "\n",
+    "The model checkpoint can be loaded directly from huggingface, and the data\n",
+    "directory points to the directory we created in the steps above.\n",
+    "\n",
+    "Note that the normalization parameters for the data module need to be \n",
+    "adapted based on the band groups that were selected as partial input. The\n",
+    "full set of normalization parameters can be found [here](https://github.com/Clay-foundation/model/blob/main/src/datamodule.py#L108)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d89e0135-9473-4f76-9f09-e4e295dd51c9",
+   "metadata": {},
+   "source": [
+    "### Load the model and set up the data module"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "301ee2db-c5fc-4628-b837-12e6ea477415",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Total number of chips: 12\n"
+     ]
+    }
+   ],
+   "source": [
+    "DATA_DIR = \"data/minicubes\"\n",
+    "CKPT_PATH = \"https://huggingface.co/made-with-clay/Clay1/resolve/main/Clay_v0.1_epoch-24_val-loss-0.46.ckpt\"\n",
+    "\n",
+    "# Load model\n",
+    "rgb_model = CLAYModule.load_from_checkpoint(\n",
+    "    CKPT_PATH,\n",
+    "    mask_ratio=0.0,\n",
+    "    band_groups={\"rgb\": (2, 1, 0), \"nir\": (3,)},\n",
+    "    bands=4,\n",
+    "    strict=False,  # ignore the extra parameters in the checkpoint\n",
+    ")\n",
+    "# Set the model to evaluation mode\n",
+    "rgb_model.eval()\n",
+    "\n",
+    "\n",
+    "# Load the datamodule, with the reduced set of\n",
+    "class ClayDataModuleRGB(ClayDataModule):\n",
+    "    MEAN = [\n",
+    "        1369.03,  # red\n",
+    "        1597.68,  # green\n",
+    "        1741.10,  # blue\n",
+    "        2858.43,  # nir\n",
+    "    ]\n",
+    "    STD = [\n",
+    "        2026.96,  # red\n",
+    "        2011.88,  # green\n",
+    "        2146.35,  # blue\n",
+    "        2016.38,  # nir\n",
+    "    ]\n",
```

Comment:
Should we document the mean/std values for all the bands (not just RGB+NIR) somewhere for v0? At least until we resolve #94?

Reply:
Added docs for this in https://github.com/Clay-foundation/model/pull/152